apps/docs-snippets/src/guide/types/numbers.test.ts

```diff
@@ -0,0 +1,73 @@
+import { toBigInt } from 'ethers';
```

Comment:
I believe we were striving to eliminate `ethers`; has that changed? 🤔

Reply:
I saw this also, however as it is in `doc-snippets` therefore not included in the `fuels` build so will not impact tree shakeability, which was one of the reasons for us trying to eliminate it. Also one of the intentions here was to show conversions between `ethers` `BigInt` and our `BigNum`, so for me it is okay here.

Reply:
Right @danielbate, and it's just a dev dependency to demonstrate the conversion.